.github/workflows/Shared-Verify-Components.yml

```diff
@@ -57,3 +57,5 @@ jobs:
         mvn verify org.sonarsource.scanner.maven:sonar-maven-plugin:sonar -Dsonar.organization=riseclipse \
         -Dsonar.host.url=http://sonarcloud.io -Dsonar.login=${{ secrets.SONAR_TOKEN }} \
         -Dsonar.projectKey=riseclipse_${{ github.event.repository.name }}
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why is-it needed?

Reply:
It doesn’t seem to work without, see this action run: https://github.com/riseclipse/riseclipse-main/runs/6008778025

Reply:
Then it must be added also in the tool workflow

Reply:
Done